PythonAPI/pycocotools/coco.py

```diff
@@ -62,6 +62,14 @@
 elif PYTHON_VERSION == 3:
     from urllib.request import urlretrieve
 
+
+def _isArrayLike(obj):
+    if hasattr(obj, '__iter__') and hasattr(obj, '__len__'):
```

Comment:
Thanks for the input. I like the idea. Could we simplify the function like:
`
return hasattr(obj, '__iter__') and hasattr(obj, '__len__')
`

Reply:
Not a bad idea. I just did that.